src/GOATalgorithm.jl

```diff
@@ -0,0 +1,152 @@
+#flat doubly stochastic matrix, starting point
+"""
+    _flat_doubly_stochastic(n)
+
+Given the dimension 'n', return the flat doubly stochastic matrix with number of rows and columns equals to 'n'.
+The flat doubly stochastic matrix is the barycenter of doubly stochastic matrices.
+"""
+_flat_doubly_stochastic(n::Int64)=ones(n) * ones(n)' / n
+
+#gradient of the objective function
+"""
+    _gradient(A,B,P_i)
+
+Given the adjacency matrices A and B, return the gradient evaluated in P_i of the function f(P)=-tr(A*P*B'*P').
```

Comment:
Here and in the rest of the docstrings also, maybe it looks nicer wrapping the expressions in LaTeX delimiters.